### DISCUSSION TITLE
2010/12/21/%E5%AE%B6%E8%B0%B1

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#308] [Announcements]
# 2010/12/21/%E5%AE%B6%E8%B0%B1

周末回无锡爸妈家走了一趟，看到了我家的家谱。

https://blog.qizhen.xyz/2010/12/21/%E5%AE%B6%E8%B0%B1

**Comment by ruanqizhen** [comment]
之前博客下的留言：

aprite
2011年01月9日 21:53 编辑
陈留阮姓，阮籍阮咸都是啊。

回复
